changelog/2022/march.rst

```diff
@@ -0,0 +1,12 @@
+--------------------------------------------------------------------------------
+                                      New                                       
+--------------------------------------------------------------------------------
+
+
+--------------------------------------------------------------------------------
+                                      Fix                                       
+--------------------------------------------------------------------------------
+
+* junos
+    * Modified ping.py
+        * Added routing instance support
```

Comment:
Please put this changelog in the 2022/undistributed folder, the release engineer will handle putting it in this file.